### PR TITLE
add a text link on front page

### LINKED ISSUE #61: Blog text link on front page
## Description

Kat's copy asks for a text link: read more about this project (goes to blog)  LINK. It needs to appear on front page under the paragraph.

Plan
- [x] update link to match blog post (at bottom of comments)

## Acceptance Criteria
- [x] The text description paragraph contains a link to the blog post.

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #61 

## Description

- Added text link on front page

🐧

### CODE REVIEW

File: src/Views/Content.elm [hunk 1]
```diff
@@ -38,13 +38,15 @@ view model =
...
-                        that app could be used by different organisations to encourage people to use a helpline or web chat or text service. We 
-                        believe that reassuring people that their fears and challenges are shared will help them take early steps on a journey of 
+                        [ text """In 2017, The Haven, Wolverhampton, Neontribe and women who had been through domestic violence co-designed a
+                        web app to encourage other women in that situation to seek support and advice. We want to find out if the structure of
+                        that app could be used by different organisations to encourage people to use a helpline or web chat or text service. We
+                        believe that reassuring people that their fears and challenges are shared will help them take early steps on a journey of
                         change. Explore the app and then take our survey to tell us what you think.
                         """
                         ]
+
```
Comment (katjam) [inline src/Views/Content.elm]: Remove the extra line? if you can work out how to checkout back into the branch. if not no worries!

**Review by katjam — CHANGES_REQUESTED** [review]
Request line removal as an exercise.

**Comment by HedgehogHolly** [comment]
@katjam updated!

**Review by katjam — APPROVED** [review]
:field_hockey: Nice

**Pull request merged**